doc/Settings.md

```diff
@@ -1,230 +1,259 @@
+In case you are suspecious that the update might not have worked, you can use this section of the admin panel to check the situation.
 
-	$a->config['system']['directory'] = 'http://dir.friendi.ca';
+## Inspect Queue
 
-Developer Settings
----
+In the admin panel summary there are two numbers for the message queues.
+The second number represents messages which could not be delivered and are queued for later retry.
+If this number goes sky-rocking you might ask yourself which receopiant is not receiving.
 
-### Debugging
-Most useful when debugging protocol exchanges and tracking down other communications issues. 
+Behind the inspect queue section of the admin panel you will find a list of the messages that could not be delivered.
+The listing is sorted by the receipiant name so identifying potential broken communication lines should be simple.
+These lines might be broken for various reasons.
+The receiving end might be off-line, there might be a high system load and so on.
 
-Config:
+Don't panic!
+Friendica will not queue messages for all time but will sort out *dead* nodes automatically after a while and remove messages from the queue then.
+
+## Federation Statistics
+
+The federation statistics page gives you a short summery of the nodes/servers/pods of the decentralized social network federation your node knows.
+These numbers are not compleate and only contain nodes from networks Friendica federates directly with.
+
+## Plugin Features
 
-	$a->config['system']['debugging'] = true;
-	$a->config['system']['logfile'] = 'logfile.out';
-	$a->config['system']['loglevel'] = LOGGER_DEBUG;
+Some of the addons you can install for your Friendica node have settings which have to be set by the admin.
+All those addons will be listed in this area of the admin panels side bar with their names.
 
-Turns on detailed debugging logs which will be stored in 'logfile.out' (which must be writeable by the webserver).
-LOGGER_DEBUG will show a good deal of information about system activity but will not include detailed data.
-You may also select LOGGER_ALL but due to the volume of information we recommend only enabling this when you are tracking down a specific problem.
-Other log levels are possible but are not being used at the present time. 
+## Logs
 
-Please be aware that turning on the logging can fill up the disk space on your server relatively quick.
-You should take preventions with e.g. [logrotate](https://en.wikipedia.org/wiki/Log_rotation) or similar tools.
+The log section of the admin panel is seperated into two pages.
+On the first, following the "log" link, you can configure how much Friendica shall log.
+And on the second you can read the log.
 
-###PHP error logging
+When you are using the default configuration for Apache, you should choose a name for the logfile ending in ``.log`` or ``.out``.
+But don't call it ``friendica.log`` (see [issue 2209](https://github.com/friendica/friendica/issues/2209)).
```

Comment:
I think it is a security rule to only have stuff in your web root that really should be accessed from the web server.